src/pages/using-spark/guides/content-style.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Content Style
+---
+
+import {
+  SprkDivider,
+} from '@sparkdesignsystem/spark-react';
+
+<h1 class="sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
+  Content Style
+</h1>
+
+UX Writers, designers and developers can use the **Content Style Guides** as a instruction to writing style, usage, text for copy, documentation, reference information, and copy for all Rocket Products digital experiences.
```

Comment:
Is "content style guides" supposed to be a link? the word *the* implies this is a document somewhere.

Reply:
is "the" supposed to be "this" ?

Reply:
I agree with RV. We're on the page, so it feels like it should be "this Content Style Guide".